test/checks/media/description.js

```diff
@@ -40,7 +40,6 @@ describe('description', function() {
 			shadow.innerHTML = '<video><slot></slot></video>';
 
 			var checkArgs = checkSetup(node, {}, 'video');
-			axe.log(checkArgs);
```

Comment:
This commit should have `test` in the scope, and is unrelated to this PR. But it kept irritating me when running tests.